test/DynamoCoreWpfTests/RecordedTests.cs

```diff
@@ -3827,6 +3829,1109 @@ public void Lacing_Deffect_5759()
+        }
+
+
+        [Test, Category("Failure")]
+        public void MAGN_7348_CreateList()
+        {
+            // Check List
+            // Scenario
+            //  a) By connecting node, 
+            //  b) check List.Create
+            // http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7348
+
+            preloadGeometry = true;
+            RunCommandsFromFile("MAGN_7348_CreateList.xml", (commandTag) =>
+            {
+                var workspace = ViewModel.Model.CurrentWorkspace;
+
+                if (commandTag == "FirstRun")
+                {
+                    NodeModel node = ViewModel.Model.CurrentWorkspace.NodeFromWorkspace
+                       ("c9bcb187-5eb7-4931-9a88-aee2ae40293b");// check List.Create
+                    Assert.AreEqual(ElementState.Active, node.State);
+                    AssertPreviewCount("c9bcb187-5eb7-4931-9a88-aee2ae40293b", 3);
+                    var list = GetFlattenedPreviewValues("c9bcb187-5eb7-4931-9a88-aee2ae40293b");
+
+                    Assert.AreEqual(list, new object[] { 42, "The answer to everything", "foo", "bar" });
+
+                    
+                }
+            });
+        }
+
+
+        [Test]
+        public void MAGN_7348_Count()
+        {
+            // Check List
+            // Scenario
+            //  a) By connecting node, 
+            //  b) check List.Combine
+            //  c) change inputs and check new result
+            //  d) incomplete input 
+            // http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7348
+
+            preloadGeometry = true;
+            RunCommandsFromFile("MAGN_7348_Count.xml", (commandTag) =>
+            {
+                var workspace = ViewModel.Model.CurrentWorkspace;
+
+                if (commandTag == "FirstRun")
+                {
+                    NodeModel node = ViewModel.Model.CurrentWorkspace.NodeFromWorkspace
+                       ("923d1dbb-9009-41af-94bf-d100fa76b9cd");// check List.Combine
+                    Assert.AreEqual(ElementState.Active, node.State);
+                    AssertPreviewCount("923d1dbb-9009-41af-94bf-d100fa76b9cd", 43);                 
+                   for (int i = 0; i < 43; i++)
+                    {
+                        string temp = "foo"+i.ToString();
+                        var list = GetPreviewValueAtIndex("923d1dbb-9009-41af-94bf-d100fa76b9cd", i);
+                        Assert.AreEqual(list.ToString(), temp);
+                    }
+                }
+                else if (commandTag == "SecondRun")
+                {
+                    AssertPreviewCount("923d1dbb-9009-41af-94bf-d100fa76b9cd", 4);// check List.Combine
+                }
+                else if (commandTag == "ThirdRun")
+                {
+                    Assert.IsNull(GetPreviewValue("923d1dbb-9009-41af-94bf-d100fa76b9cd"));// check List.Combine           
+                }
+            });
+        }
+
+
+        [Test, Category("Failure")]
```

Comment:
This test contain List.Create, which will create extra inputs and cause test failure